package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "node server",
-    "dev": "cross-env NODE_ENV=dev node server",
+    "dev": "concurrently --kill-others \"cross-env NODE_ENV=dev node server\"",
```

Comment:
```
$ npm run dev

> starter@0.1.0 dev
> concurrently --kill-others "cross-env NODE_ENV=dev node server"
```

There is probably a missing dev dependency :)

Reply:
I removed it